qir/qat/Rules/Factory.cpp

```diff
@@ -72,6 +72,8 @@ void RuleFactory::usingConfiguration(FactoryConfiguration const& config)
         optimizeResultZero();
     }
 
+    // optimizeResultComparison();
```

Comment:
Since this is commented out, the optimization is effectively disabled. Is this just a placeholder until config flags are added?

Reply:
I forgot to push the latest version.